managed-databases/document-database/api-cli/backup-and-restore.mdx

```diff
@@ -0,0 +1,109 @@
+---
+meta:
+  title: Back up Document Databases with MongoDB® tools
```

Comment:
Why not add the "restore" part to the title ?

Reply:
bc i forgot hehehe 🙏🏻